src/python/dxpy/scripts/dx.py

```diff
@@ -1737,6 +1737,7 @@ def get_output_path(obj_name, obj_class, args):
         os.unlink(output_path)
     # Here, output_path either points to a directory or a nonexistent path
     if not os.path.exists(output_path):
+        print('Creating "{f}" output directory'.format(f=output_path))
```

Comment:
Please direct these to stderr so we can keep the output clean (I don't think the output of this command is meant to be parseable necessarily but it might become so in the future).